packages/app/src/components/SourceForm.tsx

```diff
@@ -142,6 +147,85 @@ function FormRow({
   );
 }
 
+function HighlightedAttributeExpressionsFormRow({
+  control,
+  watch,
+}: TableModelProps) {
+  const databaseName = watch(`from.databaseName`, DEFAULT_DATABASE);
+  const tableName = watch(`from.tableName`);
+  const connectionId = watch(`connection`);
+
+  const {
+    fields: highlightedAttributes,
+    append: appendHighlightedAttribute,
+    remove: removeHighlightedAttribute,
+  } = useFieldArray({
+    control,
+    name: 'highlightedAttributeExpressions',
+  });
+
+  return (
+    <FormRow
+      label={'Highlighted Attributes'}
+      helpText="Expressions defining trace-level attributes which are displayed in the search side panel."
+    >
+      <Grid columns={5}>
+        {highlightedAttributes.map((field, index) => (
+          <React.Fragment key={field.id}>
+            <Grid.Col span={3} pe={0}>
+              <SQLInlineEditorControlled
+                tableConnection={{
+                  databaseName,
+                  tableName,
+                  connectionId,
+                }}
+                control={control}
+                name={`highlightedAttributeExpressions.${index}.valueExpression`}
+                disableKeywordAutocomplete
+              />
+            </Grid.Col>
+            <Grid.Col span={2} ps="xs">
+              <Flex align="center" gap="sm">
+                <Text c="gray.4">AS</Text>
+                <SQLInlineEditorControlled
+                  control={control}
+                  name={`highlightedAttributeExpressions.${index}.alias`}
+                  placeholder="Optional Alias"
+                  disableKeywordAutocomplete
+                />
+                <UnstyledButton
+                  variant="subtle"
+                  color="gray.4"
+                  size="xs"
+                  onClick={() => removeHighlightedAttribute(index)}
+                >
+                  <i className="bi bi-trash" />
+                </UnstyledButton>
+              </Flex>
+            </Grid.Col>
+          </React.Fragment>
+        ))}
+      </Grid>
+      <Button
+        variant="subtle"
+        size="sm"
+        color="gray.4"
```

Comment:
Let's avoid overriding colors. If we want a gray button we should use the `variant="default"`: https://mantine.dev/core/button/ 

Or `variant="default"` and `color="gray"` so it resolves to light and dark themes.